frontend/cypress/e2e/reviewAgreement.cy.js

```diff
@@ -161,7 +161,7 @@ describe("agreement review workflow", () => {
             cy.get("@table-rows").eq(0).find("[data-cy='expand-row']").click();
             cy.get("@table-rows").eq(0).find("[data-cy='expand-row']").click();
             cy.get("@table-rows").eq(0).find("[data-cy='expand-row']").click();
-            cy.get("[data-cy='edit-row']").click();
+            cy.get("[data-cy='edit-row']").click().click();
             cy.get(".usa-form-group--error").should("have.length", 3);
```

Comment:
double clicks?

Reply:
Long answer: I was not able to run the current Dockerfile.azure locally so could not thoroughly compare the new vs old.  I do not think we have ever run the E2E tests against Dockerfile.azure and since I cannot run it locally I have no way to test it - but I suspect that if we could run it `reviewAgreement.cy.js` would fail as well.  This is due to the front end behaving differently when run from the local development server vs. running from the production built static files.